### DISCUSSION TITLE
Customize Nav.Menu behavior

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2713] [Ideas]
So here is the example:
https://codesandbox.io/s/beautiful-payne-kj129b?file=/src/index.js

So first, if you hover the Nav.Meni and Sidenav is not expanded, tooltip will appear. If you click it, a little menu will appear.
What i would like to do. First, on hover instead of showing tooltip, i want to show that little menu. Secondly,  i want Nav.Menu to be <a> tag.

The case: Users do not want to always press Nav.Menu to open the menu, it is more convenient just to hover it.
Also, there can be the main Nav.Item, and it will be really good if user click Nav.Menu and it triggers main Nav.Item

**Comment by myNameIsDu** [comment]
Hi guy, The example is wrong (:

**Reply by abdullaev388** [reply]
Yes, sorry, here is it: https://codesandbox.io/s/beautiful-payne-kj129b?file=/src/index.js

**Reply by abdullaev388** [reply]
Does Nav provide such functionality or i have to write my own sidebar ?(
